applications/IgaApplication/tests/NodeSurfaceGeometry3DTests.py

```diff
@@ -0,0 +1,142 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.IgaApplication import *
```

Comment:
Please don't import with *. Now I am confuse

Reply:
I want the same luxury as the structural-mechanic guys :stuck_out_tongue_winking_eye:

Reply:
I am not very supportive with that...

Reply:
Thats not luxury but laziness ;)
I will clean this up in structmech at some point ...